packages/account-v2/src/containers/PersonalDetailsForm/SupportProfessionalClient.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useActiveTradingAccount, useAuthentication } from '@deriv/api-v2';
+import { Checkbox, InlineMessage, Text } from '@deriv-com/ui';
+import { AUTH_STATUS_CODES } from '../../constants/constants';
+
+export const SupportProfessionalClient = () => {
+    const { data: activeAccount } = useActiveTradingAccount();
+    const { data: authenticationStatus } = useAuthentication();
+
+    const isPoaVerified = authenticationStatus?.poa_status === AUTH_STATUS_CODES.VERIFIED;
+    const isPoiVerified = authenticationStatus?.poi_status === AUTH_STATUS_CODES.VERIFIED;
+
+    const isAccountVerified = isPoaVerified && isPoiVerified;
+    const isVirtual = activeAccount?.is_virtual;
+    return (
+        <div className='lg:max-w-[400px]'>
+            <div className='py-10'>
+                <Text as='p' className='py-10' size='sm'>
+                    By default, all Deriv.com clients are retail clients but anyone can request to be treated as a
```

Comment:
**NIT:** I think we can wither put all these statements in a list and render to prevent retyping <Text/> ?